Traducir.Core/Services/SOStringService.cs

```diff
@@ -596,5 +600,52 @@ Update Strings
             }
             return _strings;
         }
+
+        public async Task<bool> DeleteSuggestionAsync(int suggestionId, int userId)
+        {
+            var suggestion = await GetStringSuggestionByIdAsync(suggestionId);
```

Comment:
there's no need to get it in order to verify if the user is the owner. On the SQL query, you can add `And CreatedById = @userId`.

Then, you can check what `ExecuteAsync` returns. It returns the number of modified rows, so if it's 0, you know there was no suggestion with that id and userId.

You can then spare this query to the db.